src/Microsoft.AspNet.Mvc.Core/Formatters/JsonOutputFormatter.cs

```diff
@@ -16,13 +17,23 @@ public class JsonOutputFormatter : OutputFormatter, IJsonOutputFormatter
         private JsonSerializerSettings _serializerSettings;
 
         public JsonOutputFormatter()
```

Comment:
Ok, I'll leave the default ctor public and you guys can decide how to deal with it.